src/lib/io/include/sol-iio.h

```diff
@@ -72,7 +72,7 @@ typedef struct sol_iio_config {
 #define SOL_IIO_CONFIG_API_VERSION (2)
     uint16_t api_version; /**< The API version */
 #endif
-    const char *trigger_name; /**< Name of IIO trigger to be used on this device. If NULL or empty, will try to use device current trigger. If device has no current trigger, will create a 'sysfs_trigger' and use it. */
+    const char *trigger_name; /**< Name of IIO trigger to be used on this device. Set to hrtimer:<trigger name> if want to use hrtimer trigger. If NULL or empty, will try to use device current trigger. If device has no current trigger, will create a sysfs or hrtimer trigger and use it. */
```

Comment:
Please, note that generating docs fails with these `<` and `>` - see on semaphore logs.